packages/fx-core/src/plugins/resource/identity/utils/telemetryUtil.ts

```diff
@@ -1,59 +0,0 @@
-import { PluginContext } from "@microsoft/teamsfx-api";
-import { Constants, Telemetry } from "../constants";
-
-export class TelemetryUtils {
```

Comment:
we still need telemetry to collect usage data for identity plugin. We can check our existing dashboard and see what kind of telemetry is required to continue report the data in dashboard.